packages/twenty-front/src/modules/object-record/hooks/useFindDuplicateRecords.ts

```diff
@@ -0,0 +1,79 @@
+import { useMemo } from 'react';
+import { useQuery } from '@apollo/client';
+
+import { useObjectMetadataItem } from '@/object-metadata/hooks/useObjectMetadataItem';
+import { ObjectMetadataItemIdentifier } from '@/object-metadata/types/ObjectMetadataItemIdentifier';
+import { useMapConnectionToRecords } from '@/object-record/hooks/useMapConnectionToRecords';
+import { ObjectRecord } from '@/object-record/types/ObjectRecord';
+import { ObjectRecordConnection } from '@/object-record/types/ObjectRecordConnection';
+import { useSnackBar } from '@/ui/feedback/snack-bar-manager/hooks/useSnackBar';
+import { logError } from '~/utils/logError';
+
+import { ObjectRecordQueryResult } from '../types/ObjectRecordQueryResult';
+
+export const useFindDuplicateRecords = <T extends ObjectRecord = ObjectRecord>({
```

Comment:
There is no pagination now. We might think of introducing it later. I dont think that showing more than just a few duplicates makes sense for the current use case though, so i didnt bother